apps/frontend/src/app/app.jsx

```diff
@@ -0,0 +1,17 @@
+import { Routes, Route, Navigate } from "react-router-dom";
+import SignIn from "app/pages/Auth/SignIn";
+import SignUp from "app/pages/Auth/SignUp";
+import Main from "app/components/Main";
+function App() {
+  return (
+    <div className="App">
+      <Routes>
+        <Route path="/sign-up" element={<SignUp />} />
```

Comment:
@Nazi-pikachu please create the constants folder for all the routes

Reply:
resolved